src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+
+    private StringBuilder stringBuilder = new StringBuilder();
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(toFileName);
+        if (file.exists()) {
+            file.delete();
+        }
+        String[] arrayFromFile = readFromFile(fromFileName);
+        String[] arrayToFile = {"supply,", "buy,", "result,"};
+        int tempForSupply = 0;
+        int tempForBuy = 0;
+
+        for (int i = 0; i < arrayFromFile.length; i++) {
+            if (arrayFromFile[i].substring(0, arrayFromFile[i].indexOf(",")).equals("supply")) {
+                tempForSupply += Integer.parseInt(arrayFromFile[i]
+                        .substring(arrayFromFile[i].indexOf(",") + 1, arrayFromFile[i].length()));
+            } else {
+                tempForBuy += Integer.parseInt(arrayFromFile[i]
+                        .substring(arrayFromFile[i].indexOf(",") + 1, arrayFromFile[i].length()));
+            }
+        }
+        arrayToFile[0] += String.valueOf(tempForSupply);
+        arrayToFile[1] += String.valueOf(tempForBuy);
+        arrayToFile[2] += String.valueOf(tempForSupply - tempForBuy);
```

Comment:
Indexes are magic numbers, should be constant with meaningful name
```suggestion
        arrayToFile[0] += String.valueOf(tempForSupply);
        arrayToFile[1] += String.valueOf(tempForBuy);
        arrayToFile[2] += String.valueOf(tempForSupply - tempForBuy);
```